docs/book/src/Good Practices

```diff
@@ -0,0 +1,24 @@
+# Good Practices
+
+## What is "Reconciliation" in Operators?
+
+The reconciliation is a continuos  loop that performs necessary actions on the current state to ensure that the Custom Resource reaches the desired state as specified by the user.
```

Comment:
Also, we need to improve this text accordingly. 

Please, add the k8s docs here and make a reference: https://kubernetes.io/docs/concepts/extend-kubernetes/operator/